examples/non-monorepo/.gitignore

```diff
@@ -0,0 +1,39 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# next.js
+/.next/
+/out/
+
+# production
+/build
+
+# misc
+.DS_Store
+*.pem
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+.pnpm-debug.log*
+
+# local env files
+.env*.local
+
+# vercel
+.vercel
+
+# typescript
+*.tsbuildinfo
+next-env.d.ts
+
+# turbo
+.turbo
```

Comment:
Can we shave this down? E.g. `.pem` and `/coverage` seems unnecessary for the state of this example as it stands, at least

Reply:
This is the template from `npx create-next-app@latest`, and I generally prefer leaving the generated outputs as is as much as possible.